modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_portlet-forms-admin.scss

```
@@ -10,6 +10,13 @@
 	}
 }
 
+#translationManager {
+	width: 70%;
+	@media screen and (max-width: 1180px) {
+		width: 50%;
+	}
+}
+
```

Hello @kildarep, good job! Sorry for the very late review. There are just two adjustments:

- We prefer to use classes instead of ids for styling.
- If the screen is very little, when we click in a language it still appears over the sidebar, so I´d put a z-index equals to 0 to keep it bellow the sidebar.
So, the classes that also work for this case would be:

```suggestion
.ddm-translation-manager {
	.lfr-translationmanager{
		width: 70%;
		z-index: 0;
		@media screen and (max-width: 1180px) {
			width: 50%;
		}
	}	
}
```

Please, could you do this change when you have some time? 
Thanks!

---

Hi, I will make the requested changes. Thanks!